### PR TITLE
SNOW-1011780: `spcs service set/unset <properties...>` commands

### PR DESCRIPTION
**Pull request by sfc-gh-davwang**
### Pre-review checklist
   * [x] I've confirmed that instructions included in README.md are still correct after my changes in the codebase.
   * [x] I've added or updated automated unit tests to verify correctness of my new code.
   * [x] I've added or updated integration tests to verify correctness of my new code.
   * [ ] I've confirmed that my changes are working by executing CLI's commands manually.
   * [x] I've confirmed that my changes are up-to-date with the target branch.
   * [x] I've described my changes in the release notes.
   * [x] I've described my changes in the section below.

### Changes description
Added commands for spcs compute-pool set --<property1> <value1> ... and spcs compute-pool unset <name> --<property1>  .... These commands allow you to set values or reset to default values for properties of an existing compute pool, equivalent to ALTER COMPUTE POOL <name> SET/UNSET .... Added unit and integration tests for these commands.

### CODE REVIEW

File: tests/__snapshots__/test_help_messages.ambr [hunk 1]
```diff
@@ -3008,59 +3008,66 @@
...
+  │                                                             Integrations(EA… │
+  │                                                             that the service │
+  │                                                             can access. This │
+  │                                                             option may be    │
+  │                                                             specified        │
+  │                                                             multiple times   │
+  │                                                             for multiple     │
+  │                                                             EAIs.            │
+  │                                                             [default: None]  │
+  │    --query-wareho…                         TEXT             Warehouse to use │
```
Comment (sfc-gh-yiwang) [inline tests/__snapshots__/test_help_messages.ambr:3050]: Is the `--query-wareho…`  expected?

File: src/snowflake/cli/plugins/spcs/services/manager.py [hunk 2]
```diff
@@ -91,3 +92,53 @@ def upgrade_spec(self, service_name: str, spec_path: Path):
...
+        comment: bool,
+    ):
+        property_pairs = [
+            ("min_instances", min_instances),
+            ("max_instances", max_instances),
+            ("query_warehouse", query_warehouse),
+            ("auto_resume", auto_resume),
+            ("comment", comment),
+        ]
+        if not any([value for property_name, value in property_pairs]):
```
Comment (sfc-gh-yiwang) [inline src/snowflake/cli/plugins/spcs/services/manager.py:138]: It's a bit tricky. Could you add a comment to indicate if the property is not specified, the value will be false?

File: tests/spcs/test_services.py [hunk 3]
```diff
@@ -327,3 +328,177 @@ def test_list_endpoints_cli(mock_list_endpoints, mock_cursor, runner):
...
+        == f"No properties specified for service '{service_name}'. Please provide at least one property to reset to its default value."
+    )
+
+
+@patch("snowflake.cli.plugins.spcs.services.manager.ServiceManager.unset_property")
+def test_unset_property_cli(mock_unset, mock_statement_success, runner):
+    cursor = mock_statement_success()
+    mock_unset.return_value = cursor
+    service_name = "test_service"
+    result = runner.invoke(
```
Comment (sfc-gh-yiwang) [inline tests/spcs/test_services.py:463]: Do we have a way to disallow customer setting property value in unset command?
Could we add a test to verify `["spcs", "service", "unset", service_name, "--min-instances", 1]` is not allowed?

File: tests/__snapshots__/test_help_messages.ambr (hunk 1 above)
Comment (sfc-gh-davwang) [reply tests/__snapshots__/test_help_messages.ambr:3050]: yep in console works fine, I believe the test runner will just truncate names that are too long based on its window
image

**Review by sfc-gh-turbaszek — APPROVED** [review]

**Pull request merged**